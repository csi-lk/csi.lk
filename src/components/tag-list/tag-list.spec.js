import React from 'react'

import TagListComponent from './tag-list'

describe('Component: Tag List', () => {
  let props
  beforeEach(() => {
    props = {
      tags: 'test1, test2',
    }
  })
  it('renders', () => {
    const wrapper = shallow(<TagListComponent {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
