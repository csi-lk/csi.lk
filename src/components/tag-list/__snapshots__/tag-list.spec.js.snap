// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Tag List renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TagListComponent
    tags="test1, test2"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "test1",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "#",
        "test1",
      ],
      "data-tag": "test1",
    },
    "ref": null,
    "rendered": Array [
      "#",
      "test1",
    ],
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "test1",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "#",
          "test1",
        ],
        "data-tag": "test1",
      },
      "ref": null,
      "rendered": Array [
        "#",
        "test1",
      ],
      "type": "span",
    },
    Object {
      "instance": null,
      "key": "test2",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "#",
          "test2",
        ],
        "data-tag": "test2",
      },
      "ref": null,
      "rendered": Array [
        "#",
        "test2",
      ],
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
